### DISCUSSION TITLE
Not Getting Student Developer Pack

### DISCUSSION THREAD

**Discussion by Himanshu-u-rai** [community/community#56991] [GitHub Education]
the renewal of my Student Developer Pack for this year. Last year, I was able to obtain the pack using my personal email address. However, this year, I noticed that GitHub is now requiring a college-provided email address for verification. Unfortunately, my college does not offer student email addresses, and I am currently in my third year of graduation. Any Help will be Appreciated.

**Comment by Desiders** [comment]
Have a verifiable school-issued email address or upload documents that prove your current student status. Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#requirements

You can choose any option from those listed for confirmation, not just mail.

**Comment by tuves** [comment]
Welcome to the GitHub Community, @HimanshuRai114, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Desiders** [comment]
This issue seems can be closed, @HimanshuRai114

**Reply by CoolGenius-123** [reply]
I am also facing the same issue I have uploaded the required documents as well but even after that they are rejecting that and asking for college email please help I am in final year and needs urgently Student pack for my placements .

**Comment by mster1222** [comment]
> the renewal of my Student Developer Pack for this year. Last year, I was able to obtain the pack using my personal email address. However, this year, I noticed that GitHub is now requiring a college-provided email address for verification. Unfortunately, my college does not offer student email addresses, and I am currently in my third year of graduation. Any Help will be Appreciated.
